src/intl/IntlListFormat.cpp

```diff
@@ -0,0 +1,316 @@
+#if defined(ENABLE_ICU) && defined(ENABLE_INTL)
+/*
+ * Copyright (c) 2021-present Samsung Electronics Co., Ltd
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+#include "Escargot.h"
+#include "runtime/Context.h"
+#include "runtime/ExecutionState.h"
+#include "runtime/Value.h"
+#include "runtime/VMInstance.h"
+#include "runtime/ArrayObject.h"
+#include "runtime/ErrorObject.h"
+#include "runtime/IteratorObject.h"
+#include "Intl.h"
+#include "IntlListFormat.h"
+
+namespace Escargot {
+
+static const char* const intlListFormatRelevantExtensionKeys[0] = {};
+static size_t intlListFormatRelevantExtensionKeysLength = 0;
```

Comment:
It seems that `intlListFormatRelevantExtensionKeys` is used only once in this file.
Is there any reason that it should be allocated in the global scope?

Reply:
What a nice catch! I will fix it!